wagtailnhsukfrontend/mixins.py

```diff
@@ -23,3 +26,40 @@ class ReviewDateMixin(models.Model):
     class Meta:
 
         abstract = True
+
+
+class HeroMixin(models.Model):
+    """
+    A Mixin to add the model for Hero components.
+
+    """
+
+    toggle_hero_text = models.BooleanField(default=True)
```

Comment:
Instead of toggles, you could just have optional fields.
```
{% if page.hero_text and not page.hero_image %}
```

Still need the validation to make sure either hero_text or hero_image is defined